package.json

```diff
@@ -15,57 +15,57 @@
     "build": "node ./scripts/build && node dist/main.js"
   },
   "dependencies": {
-    "array.prototype.find": "^1.0.0",
-    "autoprefixer-loader": "^2.0.0",
-    "babel-core": "^5.4.7",
-    "babel-eslint": "^3.1.10",
-    "babel-jest": "^5.2.0",
-    "babel-loader": "^5.1.3",
-    "babel-runtime": "^5.4.7",
-    "body-parser": "^1.12.4",
-    "btoa": "^1.1.2",
+    "array.prototype.find": "1.0.0",
+    "autoprefixer-loader": "2.0.0",
+    "babel-core": "5.4.7",
+    "babel-eslint": "3.1.10",
+    "babel-jest": "5.2.0",
+    "babel-loader": "5.1.3",
+    "babel-runtime": "5.4.7",
+    "body-parser": "1.12.4",
+    "btoa": "1.1.2",
     "contentful": "1.1.1",
-    "css-loader": "^0.14.4",
-    "debug": "^2.2.0",
-    "es6-promise": "^2.1.1",
-    "escape-html": "^1.0.1",
-    "esprima-loader": "^0.9.0",
-    "express": "^4.12.4",
-    "extract-text-webpack-plugin": "^0.8.1",
-    "file-loader": "^0.8.4",
-    "flux": "^2.0.3",
-    "glob": "^5.0.10",
-    "immutable": "^3.7.3",
-    "isomorphic-fetch": "^2.1.0",
-    "marked": "^0.3.5",
+    "css-loader": "0.14.4",
+    "debug": "2.2.0",
+    "es6-promise": "2.1.1",
+    "escape-html": "1.0.1",
+    "esprima-loader": "0.9.0",
+    "express": "4.12.4",
+    "extract-text-webpack-plugin": "0.8.1",
+    "file-loader": "0.8.4",
+    "flux": "2.0.3",
+    "glob": "5.0.10",
+    "immutable": "3.7.3",
+    "isomorphic-fetch": "2.1.0",
+    "marked": "0.3.5",
     "node-noop": "0.0.1",
-    "normalize.css": "^3.0.3",
-    "object.assign": "^3.0.0",
-    "react": "^0.13.3",
-    "react-doc-meta": "^0.2.0",
+    "normalize.css": "3.0.3",
+    "object.assign": "3.0.0",
+    "react": "0.13.3",
```

Comment:
haha, yeah for sure. 0.14.2  is currently breaking our nav though, so just a temporary fix until we have time to update all the other dependencies